src/core/variable_mx.jl

```diff
@@ -68,7 +68,7 @@ function variable_mx_real(model::JuMP.Model, indices::Array{T,1}, N::Int, M::Int
     dict_mat_vars_temp = Dict{T, Array{Any, 2}}([(i,fill(missing,N,M)) for i in indices])
     for n in 1:N
         for m in 1:M
-            varname = isempty(prefix) ? "$(n)$(m)_$name" : "$(prefix)_$(n)$(m)_$name"
+            varname = isempty(prefix) ? "$(name)_$(n)$(m)" : "$(prefix)_$(n)$(m)"
```

Comment:
`$(prefix)_$(n)$(m)` should be `$(prefix)_$(name)_$(n)$(m)`

prefix is usually the network number

Reply:
correct in the other functions though, typo probably

Reply:
thanks, changed now